src/components/UI/market/SelectCategory.tsx

```diff
@@ -60,6 +58,7 @@ const SelectCategory: React.FC<SelectCategoryProps> = ({
           {category}
         </Category>
       ))}
+      <button onClick={() => fetchNextPage()}>다음 페이지</button>
```

Comment:
아래처럼 simplify 할 수 있을듯

```suggestion
      <button onClick={fetchNextPage}>다음 페이지</button>
```